src/main/resources/db/migration/V20220327_117__HMAN-117-Drop_non-null_chr_constraints.sql

```diff
@@ -0,0 +1,12 @@
+ALTER TABLE public.case_hearing_request ALTER COLUMN auto_list_flag DROP NOT NULL;
```

Comment:
Changes in this file due to audit approach for DELETE, where none of these fields will have values.